assert/assertions_test.go

```diff
@@ -1029,30 +1029,50 @@ func TestContainsNotContains(t *testing.T) {
 	type A struct {
 		Name, Value string
 	}
-	list := []string{"Foo", "Bar"}
+	list := []string{"Hello", "World"}
+
+	byteSliceList := [][]byte{
+		[]byte("Hello"), []byte("World"),
+	}
 
 	complexList := []*A{
 		{"b", "c"},
 		{"d", "e"},
 		{"g", "h"},
 		{"j", "k"},
 	}
-	simpleMap := map[interface{}]interface{}{"Foo": "Bar"}
+
+	mixedList := []interface{}{"Hello World", 5, 2, []byte("Goodbye")}
+
+	simpleMap := map[interface{}]interface{}{"Hello": "World"}
+	strMap := map[string]string{"Hello": "World"}
 	var zeroMap map[interface{}]interface{}
 
 	cases := []struct {
 		expected interface{}
 		actual   interface{}
 		result   bool
 	}{
-		{"Hello World", "Hello", true},
-		{"Hello World", "Salut", false},
-		{list, "Bar", true},
-		{list, "Salut", false},
+		{"Hello World", "World", true},
+		{"Hello World", "Earth", false},
+		{[]byte("Hello World"), []byte("World"), true},
+		{[]byte("Hello World"), []byte("Earth"), false},
+		{[]byte("ABC"), 'B', false},
+		{[]byte("ABC"), rune('B'), false},
+		{[]byte("ABC"), byte('B'), true},
```

Comment:
I feel like these should be added

```suggestion
		{[]byte("ABC"), byte('B'), true},
		{[]byte("ABC"), []byte("B"), true},
		{[]byte("ABC"), []byte{66}, true},
		{[]byte("ABC"), []byte("AB"), true},
		{[]byte("ABC"), []byte{65, 66}, true},
```

https://go.dev/play/p/XTcbu5yNaRh

Reply:
In fact, I commented too fast.

This is the unit tests and not the godoc.

So here the equivalent notation such as []byte("AB") and []byte{65,66}  are not needed here

I feel like this would be enough, even if I conceed they are somehow covered by the hello world unit tests already present in the file.

```suggestion
		{[]byte("ABC"), byte('B'), true},
		{[]byte("ABC"), []byte("B"), true},
		{[]byte("ABC"), []byte("AB"), true},
```